build.gradle.kts

```diff
@@ -71,6 +75,51 @@ tasks.shadowJar {
     }
 }
 
+val verifyKotlinCompilerDownload by tasks.creating(Verify::class) {
```

Comment:
```suggestion
val verifyKotlinCompilerDownload by tasks.registering(Verify::class) {
```

Reply:
I was going to use task configuration avoidance but saw your existing tasks in the build file didn't :)

But I'll update this for the new tasks 👍

Reply:
hehe :D yes I copy pasted chunks of detekt's build file back in january without configuration avoidance :)